docs/plans/2026-03-03-bank-transaction-linked-info-design.md

```
@@ -0,0 +1,63 @@
+1. `pending_outflows` via reverse FK `linked_bank_transaction_id` — fields: `vendor_name`, `notes`, `reference_number`, `payment_method`, `status`
+2. `expense_invoice_uploads` via FK `expense_invoice_upload_id` — fields: `vendor_name`, `invoice_number`
+
+Add `expense_invoice_upload_id` to the SELECT column list (already exists on table, currently excluded from list query).
+
+## UI Design
+
+### Desktop Row (MemoizedTransactionRow)
+
+When a transaction has `linked_outflow` or `expense_invoice_upload`:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)